homeassistant/components/simplisafe/config_flow.py

```diff
@@ -82,16 +82,31 @@ async def async_step_user(
             )
 
         errors = {}
-        session = aiohttp_client.async_get_clientsession(self.hass)
 
+        if len(user_input[CONF_AUTH_CODE]) != 45:
```

Comment:
Maybe create a variable and use that variable on subsequent checks/uses
```suggestion
        if len(auth_code := user_input[CONF_AUTH_CODE]) != 45:
```